cmake/cmaize/toolchain/toolchain.cmake

```diff
@@ -108,78 +146,19 @@ cpp_class(Toolchain)
         # Autopopulate some values from the CMake environment
         cpp_map(SET "${auto_opt_map}" "CMAKE_C_COMPILER" "${CMAKE_C_COMPILER}")
```

Comment:
Shouldn't you be looping over `_cmaize_Toolchain_autopopulated_variable_names`?

Reply:
Yes, good call.